indy_node/server/batch_handlers/domain_batch_handler.py

```diff
@@ -0,0 +1,69 @@
+from collections import Callable
+
+import base58
+
+from plenum.server.batch_handlers.domain_batch_handler import DomainBatchHandler as PDomainBatchHandler
+from plenum.server.database_manager import DatabaseManager
+
+
+class DomainBatchHandler(PDomainBatchHandler):
+    def __init__(self, database_manager: DatabaseManager):
+        super().__init__(database_manager)
+        self.post_batch_creation_handlers = []
```

Comment:
I think we should avoid registering `post_xxx_handlers` here. The idea is to instantiate and new `BatchHandler` if we want a handler, and register it in `WriteRequestManager`, not in `DomainBatchHandler`.

Reply:
fixed